bin/utils/descriptions/generate_processing_page.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from flask.json import htmlsafe_dumps
+from jinja2 import Environment, FileSystemLoader
+from sqlalchemy.orm import load_only
+
+# Used only by imports in the next block
```

Comment:
are those strange imports actually needed? iirc i tried it once without them and it still worked..